scripts/west_commands/build.py

```diff
@@ -78,20 +87,17 @@ def do_add_parser(self, parser_adder):
             self.name,
             help=self.help,
             formatter_class=argparse.RawDescriptionHelpFormatter,
-            description=self.description)
+            description=self.description,
+            usage='''west build [-h] [-b BOARD] [-d BUILD_DIR]
```

Comment:
Not sure how indentation is treated, @mbolivar can probably tell me if I should be reflowing this text programatically. For now I just played around until it looked identical to the one generated by argparse.

Reply:
See notes about west's main.py in my other comment.